sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample06_Transactions.md

```diff
@@ -44,6 +44,43 @@ using (var ts = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
 }
 ```
 
+### Transactions across entities
+When creating senders and receivers that should be part of a cross-entity transaction, you can specify the same `TransactionGroup` string in the options as shown below. When using transaction groups, the first entity that an operation occurs on becomes the entity through which all subsequent sends will be routed through. This enables the service to perform a transaction that is meant to span multiple entities. This means that subsequent entities that perform their first operation need to either be senders, or if they are receivers they need to be on the same entity as the initial entity through which all sends are routed through (otherwise the service would not be able to ensure that the transaction is committed because it cannot route a receive operation through a different entity). For instance, if you have SenderA and ReceiverB that are part of the same transaction group, you would need to receive first with ReceiverB to allow this to work. If you first used SenderA to send to QueueA, and then attempted to receive from QueueB, an `InvalidOperationException` would be thrown. You could still add a ReceiverA to the same transaction group after initially sending to SenderA, since they are both using the same queue. This would be useful if you also had a SenderB that you want to include as part of the transaction group (otherwise there would be no need to use transaction groups as you would be dealing with only one entity).
```

Comment:
```suggestion
When creating senders and receivers that should be part of a cross-entity transaction, you can specify the same `TransactionGroup` string in the options as shown below. When using transaction groups, the first entity that an operation is performed on (send or receive) becomes the entity through which all subsequent sends will be routed through.  This behavior allows the service to initiate a transaction that spans multiple entities.  This constraint means that other entities included in the transaction must first have a message sent to them.  Receiving must take place on the initial entity which is now where all sends are routed through. 

For example, assume that SenderA is using QueueA and ReceiverB is using QueueB.  If SenderA and ReceiverB belong to the same transaction group, you would need to receive first with ReceiverB to avoid participating in a cross-entity transaction. If SenderA publishes a message first, then ReceiverB would trigger an `InvalidOperationException`  trying to read from QueueB, because the transaction is routed through QueueA.  You would still be able to later add ReceiverA to the same transaction group to receive from QueueA, since QueueA is the entity through which the transaction is routed.  You would also be able to add SenderB to the transaction group and publish to QueueB, as Service Bus will ensure that the message is routed through QueueA and visible to the transaction.
```

Reply:
I had a difficult time understanding;  I tried rewriting the paragraph primarily to force myself to read it over a few times until I *think* that I get it.    I left it here for two reasons - first, so that you can correct me if I'm still not getting it and second, in case the rephrasing helps to illustrate where I struggled.

Reply:
Chatted offline - will update to include the example of what won't work in the sample